<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSlave</class>
 <widget class="QDialog" name="EditSlave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slave Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="selectClientGroupBox">
     <property name="title">
      <string>Select slave : </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="slaveSelectedBox">
        <property name="text">
         <string>Path to slave</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectSlaveButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="infosOriginalesGroupBox">
     <property name="title">
      <string>Source fields</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="sourceMasterIpLabel">
        <property name="text">
         <string>Master IP : </string>
        </property>
        <property name="buddy">
         <cstring>sourceMasterIpEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sourceMasterIpEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sourceMasterPortlLabel">
        <property name="text">
         <string>Master Port : </string>
        </property>
        <property name="buddy">
         <cstring>sourceMasterPortEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="sourceMasterPortEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sourceSlaveNameEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sourceSlaveNameLabel">
        <property name="text">
         <string>Slave name : </string>
        </property>
        <property name="buddy">
         <cstring>sourceSlaveNameEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="modifyClientGroupBox">
     <property name="title">
      <string>Destination fields</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="destinationMasterIpLabel">
        <property name="text">
         <string>Master IP : </string>
        </property>
        <property name="buddy">
         <cstring>destinationMasterIpEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="destinationMasterIpEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="destinationMasterPortLabel">
        <property name="text">
         <string>Master Port : </string>
        </property>
        <property name="buddy">
         <cstring>destinationMasterPortEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="destinationSlaveNameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="destinationSlaveNameLabel">
        <property name="text">
         <string>Slave name : </string>
        </property>
        <property name="buddy">
         <cstring>destinationSlaveNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="destinationSlavePasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="destinationSlavePasswordLabel">
        <property name="text">
         <string>Password : </string>
        </property>
        <property name="buddy">
         <cstring>destinationSlavePasswordEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="destinationMasterPortEdit">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>65535.000000000000000</double>
        </property>
        <property name="value">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="modifyExeButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelEditorButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>selectSlaveButton</tabstop>
  <tabstop>destinationSlaveNameEdit</tabstop>
  <tabstop>destinationMasterIpEdit</tabstop>
  <tabstop>destinationMasterPortEdit</tabstop>
  <tabstop>destinationSlavePasswordEdit</tabstop>
  <tabstop>modifyExeButton</tabstop>
  <tabstop>cancelEditorButton</tabstop>
  <tabstop>sourceMasterIpEdit</tabstop>
  <tabstop>sourceMasterPortEdit</tabstop>
  <tabstop>slaveSelectedBox</tabstop>
  <tabstop>sourceSlaveNameEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
